src/electron/views/screenshot/screenshot.tsx

```diff
@@ -5,9 +5,11 @@ import * as React from 'react';
 
 export type ScreenshotProps = {
     encodedImage: string;
-    altText: string;
+    altText?: string;
```

Comment:
Seeing the optional altText param made me expect that there wouldn't be any alt text if it wasn't passed; I think if we're going to have Screenshot be the thing with responsibility for producing the alt text, it would maybe be more clear to just omit it from the props here

Reply:
Removed altText param